### ISSUE TITLE
Is Lamda too high?

### ISSUE THREAD

**Issue by J-Calvelo** [hahnlab/CAFE5#101]
Hello. I'm doing some test runs with CAFE5 and while they all run to completion and seem to converge to the same value depending on the Gamma value, the estimated lambda ranges from 0.92 to 0.99. Is this to be expected or am I having some genome duplication or problems with the annotation that I'm not aware about?

Thanks

**Comment by hahnlab-user** [comment]
Yes, that seems much too high! Have you tried running these data in the base model (i.e. not gamma) to see what value of lambda is inferred?

**Comment by J-Calvelo** [comment]
I have, I did 10 runs with the base model and is always around 0.92407

**Comment by hahnlab-user** [comment]
Hmm, I'm not sure that should be possible. Can you please share your species tree, with branch lengths? Thanks

**Comment by J-Calvelo** [comment]
Here
(Bulinus_truncatus_GCA_021962125.1:0.200679,(Biomphalaria_straminea_gigascience:0.0811604,(Biomphalaria_glabrata_VectorBaseBB02:0.0773435,(Biomphalaria_sudanica:0.0346364,Biomphalaria_pfeifferi_SamTeam:0.0390029)0.455394:0.0382115)0.474654:0.0518471)1:0.200679);

**Comment by hahnlab-user** [comment]
I see. I think it may just be the scaling of lambda that seems strange: since most people report it per million years, branch lengths that are in different units will make lambda different. Also note that we recommend using ultrametric trees, as non-ultrametric trees such as this one are hard to interpret under our model.

**Comment by J-Calvelo** [comment]
I see, thanks
